.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+
+[*.{ts,json,js}]
```

Comment:
Should include tsx and jsx as well right?

Reply:
Oh, forgot that. Thanks!

Reply:
Also `*.html` was missing. Now it should do the job :+1: